flixel/FlxState.hx

```diff
@@ -80,6 +80,8 @@ class FlxState extends FlxGroup
 	@:noCompletion
 	var _subStateClosed:FlxTypedSignal<FlxSubState->Void>;
     
+	public function new() {super();}
```

Comment:
shouldn't be needed, by default if a constructor isn't given to an extending class it just calls super automatically

Reply:
Well, I had to add that cos the FlxGroup had a public function new with args, making every state, no matter if you don't have any, use bind